File: nodecore-spv/src/main/java/veriblock/model/LedgerValue.java
package veriblock.model;

public class LedgerValue {
    private final long availableAtomicUnits;
    private final long frozenAtomicUnits;
    private final long signatureIndex;

    public LedgerValue(Long availableAtomicUnits, Long frozenAtomicUnits, Long signatureIndex) {
        this.availableAtomicUnits = availableAtomicUnits;
        this.frozenAtomicUnits = frozenAtomicUnits;
        this.signatureIndex = signatureIndex;
    }

    public long getAvailableAtomicUnits() {
        return availableAtomicUnits;
    }

    public long getFrozenAtomicUnits() {
        return frozenAtomicUnits;
    }

    public long getSignatureIndex() {
        return signatureIndex;
    }

}

File: nodecore-spv/src/main/java/org/veriblock/spv/model/LedgerValue.kt
package org.veriblock.spv.model

data class LedgerValue(
    val availableAtomicUnits: Long,
    val frozenAtomicUnits: Long,
    val signatureIndex: Long
) {
    override fun toString(): String {
        return "available=$availableAtomicUnits frozen=$frozenAtomicUnits sigIndex=$signatureIndex"
    }
}
